eng/tools/TypeSpecValidation/package.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "type-spec-validation",
+    "version": "0.0.1",
+    "dependencies": {
+        "simple-git": "^3.16.0"
+    }
+}
```

Comment:
```suggestion
  "devDependencies": {
    "@tsconfig/node16": "^1.0.4",
    "typescript": "~5.1.3"
  }
}
```

Reply:
I think we should try to use `@tsconfig/node16` as the base for our tsconfig.json.

Reply:
I'm pretty sure we always want a `devDependency` on `typescript` to declare the version, floating with `~` since TypeScript allows breaking changes in minor version.

Reply:
Actually, let's do the minimum to get this merged, and add features in later PRs.

Tracking issue: https://github.com/Azure/azure-rest-api-specs/issues/24304